easybuild/easyconfigs/i/imkl/imkl-2023.1.0-gompi-2023a.eb

```diff
@@ -10,7 +10,7 @@ description = "Intel oneAPI Math Kernel Library"
 toolchain = {'name': 'gompi', 'version': '2023a'}
 
 # see https://software.intel.com/content/www/us/en/develop/articles/oneapi-standalone-components.html
-source_urls = ['https://registrationcenter-download.intel.com/akdlm/irc_nas/18721/']
+source_urls = ['https://registrationcenter-download.intel.com/akdlm/IRC_NAS/18721/']
 sources = ['l_onemkl_p_%(version)s.46342_offline.sh']
```

Comment:
This is the only failure remaining - and it fails with both old and new version of the URL.